pkg/ddl/ingest/checkpoint.go

```diff
@@ -396,7 +408,7 @@ func (s *CheckpointManager) afterFlush() {
 	defer s.mu.Unlock()
 	for {
 		cp := s.checkpoints[s.minTaskIDFinished]
-		if cp == nil || !cp.lastBatchRead || cp.writtenKeys < cp.totalKeys {
+		if cp == nil || !cp.lastBatchRead || cp.writtenKeys < cp.totalKeys || cp.chunksFinished < cp.chunksTotal {
```

Comment:
IF !(cp.writtenKeys < cp.totalKeys), then chunksFinished must equal to chunksTotal?
So it may always be false here?